src/core/mocks/data/data-type.data.ts

```diff
@@ -277,7 +277,15 @@ export const data: Array<DataTypeDetails> = [
 		isFolder: false,
 		propertyEditorModelAlias: 'Umbraco.RadioButtonList',
 		propertyEditorUIAlias: 'Umb.PropertyEditorUI.RadioButtonList',
-		data: [],
+		data: [
+			{
+				alias: 'itemList',
+				value: [
+					{ label: 'Label 1', key: '123' },
```

Comment:
Looks like the old backoffice uses "value" instead of "key", but "label" is fine